docs/processes/release/tasks/versions.md

```diff
@@ -92,6 +92,126 @@ Edit icuver.txt directly.
 
 ### ICU4J
 
+#### Since ICU 74
+
+Changing the version for Java starting with ICU 74 requires a few steps:
+
+1. [icu4j/main/core/src/main/java/com/ibm/icu/util/VersionInfo.java](https://github.com/unicode-org/icu/blob/main/icu4j/main/core/src/main/java/com/ibm/icu/util/VersionInfo.java)
+
+    There is a static block starting at line 501 (as of 54.1) in the source file -
+
+    <pre>
+    /**
+     * Initialize versions only after MAP_ has been created
+     */
+    static {
+        UNICODE_1_0 = getInstance(1, 0, 0, 0);
+        UNICODE_1_0_1 = getInstance(1, 0, 1, 0);
+        UNICODE_1_1_0 = getInstance(1, 1, 0, 0);
+        UNICODE_1_1_5 = getInstance(1, 1, 5, 0);
+        UNICODE_2_0 = getInstance(2, 0, 0, 0);
+        UNICODE_2_1_2 = getInstance(2, 1, 2, 0);
+        UNICODE_2_1_5 = getInstance(2, 1, 5, 0);
+        UNICODE_2_1_8 = getInstance(2, 1, 8, 0);
+        UNICODE_2_1_9 = getInstance(2, 1, 9, 0);
+        UNICODE_3_0 = getInstance(3, 0, 0, 0);
+        UNICODE_3_0_1 = getInstance(3, 0, 1, 0);
+        UNICODE_3_1_0 = getInstance(3, 1, 0, 0);
+        UNICODE_3_1_1 = getInstance(3, 1, 1, 0);
+        UNICODE_3_2 = getInstance(3, 2, 0, 0);
+        UNICODE_4_0 = getInstance(4, 0, 0, 0);
+        UNICODE_4_0_1 = getInstance(4, 0, 1, 0);
+        UNICODE_4_1 = getInstance(4, 1, 0, 0);
+        UNICODE_5_0 = getInstance(5, 0, 0, 0);
+        UNICODE_5_1 = getInstance(5, 1, 0, 0);
+        UNICODE_5_2 = getInstance(5, 2, 0, 0);
+        UNICODE_6_0 = getInstance(6, 0, 0, 0);
+        UNICODE_6_1 = getInstance(6, 1, 0, 0);
+        UNICODE_6_2 = getInstance(6, 2, 0, 0);
+        UNICODE_6_3 = getInstance(6, 3, 0, 0);
+        UNICODE_7_0 = getInstance(7, 0, 0, 0);
+    <b>
+        ICU_VERSION = getInstance(54, 1, 0, 0);
+        ICU_DATA_VERSION = ICU_VERSION;</b>
+        UNICODE_VERSION = UNICODE_7_0;
+
+        UCOL_RUNTIME_VERSION = getInstance(8);
+        UCOL_BUILDER_VERSION = getInstance(9);
+        UCOL_TAILORINGS_VERSION = getInstance(1);
+    }
+    </pre>
+
+    In the same file, starting at line 164 (as of 54.1) -
+
+    <pre>
+    /**
+     * Data version string for ICU's internal data.
+     * Used for appending to data path (e.g. icudt43b)
+     * @internal
+     * @deprecated This API is ICU internal only.
+     */
+    @Deprecated
+    public static final String <b>ICU_DATA_VERSION_PATH = "54b";</b>
+    </pre>
+
+2. When creating the final release of a major ICU version, 
+or a point release (minor version update on a maintenance branch),
+update the Maven project version for ICU4J at the root and all of the submodules
```

Comment:
Nitpick: 
  > at the root and all of the submodules

maybe say (in a bracket) that this root means "<icu_root>/icu4j" ?

Reply:
Done.